server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -627,3 +627,61 @@ pub fn router() -> Router {
             .route("attempt/msg/:msg_id/", get(list_attempts_by_msg)),
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use validator::Validate;
+
+    use super::{ListAttemptedMessagesQueryParameters, ListAttemptsByEndpointQueryParameters};
+
+    #[test]
+    fn test_list_attempted_messages_query_params_validation() {
+        let q: ListAttemptedMessagesQueryParameters = serde_json::from_str(
+            &r#"{
+                "channel": ",invalid",
+                "status": 3
```

Comment:
You can just omit it.

Reply:
Sorry, omit what?

Reply:
You don't need to include `"status"`, you can just test just `"channel"`. It's optional anyway, so just omitting is fine and makes what you are testing clearer.